src/c_api.rs

```diff
@@ -278,8 +265,6 @@ fn get_int_value(value: &Value) -> Option<c_long> {
 fn get_double_value(value: &Value) -> Option<c_double> {
     match value {
         Value::Number(n) if n.is_f64() => n.as_f64(),
-        Value::String(s) => c_double::from_str(s.as_str()).ok(),
```

Comment:
Can we keep the string->double conversion here? It helps the API be less tight-ass.

Reply:
@rafie OK.
Why only string->double? (Geo?)